edxval/tests/test_api.py

```diff
@@ -852,7 +862,9 @@ def test_get_videos_for_ids_duplicates(self):
         self.assertEqual(len(videos), 1)
 
     def test_get_videos_for_ids_sort(self):
+        """ Tests for videos """
```

Comment:
If you regenerate `pylintrc` with a relatively recent version of `edx-lint`, it shouldn't require you to provide docstrings for test methods.